src/coreclr/tools/Common/TypeSystem/CodeGen/TypeDesc.CodeGen.cs

```diff
@@ -25,5 +25,11 @@ public partial class InstantiatedType
             if (_typeDef.IsIntrinsic)
                 flags |= TypeFlags.IsIntrinsic;
         }
+
+        partial void AddComputedInlineArrayFlag(ref TypeFlags flags)
```

Comment:
Technically this should go to a new file `TypeDesc.Metadata.cs`, but maybe this is fine.

Reply:
> Technically this should go to a new file TypeDesc.Metadata.cs, but maybe this is fine.

I will not change this part then. 
Let me know if you rethink this.